pkg/render/logstorage.go

```diff
@@ -1335,5 +1351,27 @@ func (m managedClusterLogStorage) linseedExternalRolesAndBindings() ([]*rbacv1.C
 		},
 	}
 
-	return []*rbacv1.ClusterRole{secretsRole, configMapsRole}, []*rbacv1.RoleBinding{configMapBinding, secretBinding}
+	return []*rbacv1.ClusterRole{secretsRole, configMapsRole, namespacesRole}, []*rbacv1.RoleBinding{configMapBinding, secretBinding}
+}
+
+func (m managedClusterLogStorage) linseedExternalClusterRoleBinding() *rbacv1.ClusterRoleBinding {
+
+	namespacesBinding := &rbacv1.ClusterRoleBinding{
```

Comment:
Why create the binding here instead of in `linseedExternalRolesAndBindings` like we do for the other ones?